clusterloader2/pkg/measurement/common/wait_for_pvcs.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog"
+	"k8s.io/perf-tests/clusterloader2/pkg/measurement"
+	measurementutil "k8s.io/perf-tests/clusterloader2/pkg/measurement/util"
+	"k8s.io/perf-tests/clusterloader2/pkg/util"
+)
+
+const (
+	defaultWaitForPVCsTimeout       = 60 * time.Second
```

Comment:
My testing scripts set a timeout rather than use the default. I think 60 sec is a good upper bound for non stress situations.

Reply:
Well - the purpose of scale tests is actually stress, so I would rather be quite conservative in setting this timeout.

Reply:
That hard part about this is the wait time for PVCs depend heavily on the # of PVCs, since the backend can only provision disks so fast. I dont think its possible to set a good default value that wont be too low or too high. In my tests, what I do is set this to be `MAX(10 sec, NUM_PVCS * PER_PVC_SLO)`